src/app/modules/core/inbox/inbox.component.scss

```diff
@@ -0,0 +1,145 @@
+$width280: 280px;
+$backgroundwhite: #ffffff;
+$borderleftblue: #81d4fa;
+$value27px: 27px;
+$value54px: 54px;
+$value16px: 16px;
+
+:host {
```

Comment:
every component should scss should have a parent class like 
.inbox {
}

and all css should be written inside it. so that its ensured that css is only for that component. not how its done right now